src/features/home/testimonials/Testimonials.tsx

```diff
@@ -0,0 +1,149 @@
+'use client';
+
+import { Avatar, Box } from '@mui/material';
+import Image from 'next/image';
+import { EffectCoverflow, Pagination } from 'swiper/modules';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+import SectionHeading from '@/components/section-heading/SectionHeading';
+import {
+  TestimonialsAvatarNameWrapper,
+  TestimonialsCard,
+  TestimonialsCardHeading,
+  TestimonialsCardPara,
+  TestimonialsHeading,
+  TestimonialsOccupationPara,
+  TestimonialsWrapper,
+} from '@/features/home/testimonials/Testimonials.style';
+import 'swiper/css';
+import 'swiper/css/effect-coverflow';
+import 'swiper/css/pagination';
+
+const testimonialsData = [
```

Comment:
kindly move this static testimonialsData to another file and can you explain why it is marked as "use client" and some other files?

Reply:
For client-side components, we need to add use client at the top. Otherwise, it will throw an error, and the front end won't function properly